src/meson.build

```diff
@@ -51,10 +51,14 @@ foreach h: headers
   epoxy_headers += join_paths(meson.source_root(), 'include/epoxy/@0@'.format(h))
 endforeach
 
+common_ldflags = []
 if cc.get_id() == 'gcc'
   common_ldflags = [ '-Wl,-Bsymbolic', '-Wl,-z,relro', '-Wl,-z,now', ]
```

Comment:
Btw, I wanted to mention this elsewhere—aren't these options also supported when clang/icc is the compiler since these are passed to ld anyway? So this should be `cc.get_id() != 'msvc'`?